src/TPPCore.Service.Common/AspNetUtils/HttpContextExtensions.cs

```diff
@@ -8,23 +8,22 @@ namespace TPPCore.Service.Common.AspNetUtils
     public static class HttpContextExtensions
     {
         /// <summary>
-        /// Sets the appropriate HTTP headers and writes the given JSON object.
+        /// Sets the appropriate HTTP headers and writes the given string object.
         /// </summary>
-        public async static Task RespondJsonAsync(this HttpContext context, JObject doc)
+        public async static Task RespondStringAsync(this HttpContext context, string doc)
```

Comment:
This confused me for a bit. The documentation could say that it writes a "JSON-formatted string" and not just any arbitrary string since it sets the content type to be JSON.